packages/cli/src/commands/generate/auth/auth.js

```diff
@@ -147,7 +147,7 @@ export const handler = async ({ provider, force }) => {
   const tasks = new Listr(
     [
       {
-        title: 'Adding required packages...',
+        title: 'Adding required web packages...',
```

Comment:
Now can add packages to either the web or api side, since Auth0's api side benefits from having the its `AuthenticationClient` to implement the getCurrentUser.

Reply:
@peterp ok with ☝️ to add both web/api packages?